README.md

```diff
@@ -1,3 +1,9 @@
+**For bug reports, please follow the standard [Unity Bug Reporting](https://unity3d.com/unity/qa/bug-reporting) process.**
```

Comment:
I'm not sure if this is enough or we should be more clear about the fact that we will no longer accept bugs over GitHub issues — but for now, I wanted to start simple unless you folks think otherwise.